### PR TITLE
docs: add missing title to annotations index

### PR DESCRIPTION
**Pull request by itaysk**
Signed-off-by: Itay Shakury <itay@itaysk.com>

I noticed that the title annotation is missing from the table on top

### CODE REVIEW

File: docs/content/annotations.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ organizations | list of strings | A list of organizations related to the annotat
 related_resources | list of URLs | A list of URLs pointing to related resources/documentation. Read more [here](#related-resources).
 schemas | list of object | A list of associations between value paths and schema definitions. Read more [here](#schemas).
 scope | string; one of `package`, `rule`, `document`, `subpackages` | The scope on which the `schemas` annotation is applied. Read more [here](./#scope).
+title | string | A human-redable name for the annotation target. Read more [here](#title).
```
Comment (srenatus) [inline docs/content/annotations.md:43]: Thanks for fixing this! I think it should come first, though. Would you mind shuffling this up?

**Review by srenatus — APPROVED** [review]
Thanks! One nitpick 🙃

File: docs/content/annotations.md (hunk 1 above)
Comment (itaysk) [reply docs/content/annotations.md:43]: I thought so as well, but I noticed that the index table was in order with the content in the body, so I kept the same order. while I'm touching it, do you think it's worth to reorder like so:

1. scope
1. title
1. description
1. authors
1. organizations
1. schemas
7. custom

or I could just keep the order and move title to the top like you suggested.

Comment (srenatus) [reply docs/content/annotations.md:43]: Would you mind reordering? That sounds reasonable.

**Comment by itaysk** [comment]
IDK why it says that I dismissed your review, I addressed your comment :)

**Comment by itaysk** [comment]
and rebased now

**Review by srenatus — APPROVED** [review]
Thanks again

**Pull request merged**